awscli/customizations/cloudtrail.py

```diff
@@ -184,6 +186,19 @@ def _call(self, options, parsed_globals):
                 'Logs will be delivered to {bucket}:{prefix}\n'.format(
                     bucket=bucket, prefix=options.s3_prefix or ''))
 
+    def _get_policy(self, key_name):
+        try:
+            data = self.s3.GetObject(
+                bucket='awscloudtrail-policy-' + self.region_name,
+                key=key_name)
+            policy = data['Body'].read().decode('utf-8')
```

Comment:
You can just `return data['Body'].read().decode('utf-8')` instead of needing the `return policy` on line 200.